packages/datadog-plugin-bluebird/test/index.spec.js

```diff
@@ -3,3 +3,5 @@
 const assertPromise = require('../../dd-trace/test/plugins/promise')
 
 assertPromise('bluebird')
+
+assertPromise('bluebird', bluebird => bluebird.getNewLibraryCopy(), '^2.11.0 || ^3.4.1')
```

Comment:
Just realized that a test is missing for chaining, for example `bluebird.getNewLibraryCopy().getNewLibraryCopy()`. I'm also not sure if the code will handle it right now since the new class returned is not patched.

Reply:
it wouldn't handle it right now, tho i'm not sure if that use case is really used in practice? I can try to account for that though, will try to push something up before eod

Reply:
updated, lmk what you think